build/targets/legacy-lifecycle.targets

```diff
@@ -0,0 +1,158 @@
+  <Target Name="target-dir-clean">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=target-dir-clean" />
+  </Target>
+  <Target Name="build-clean">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=build-clean" />
+  </Target>
+  <Target Name="ci-deep-clean">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=ci-deep-clean" />
+  </Target>
+  <Target Name="build-compile">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=build-compile" />
+  </Target>
+  <Target Name="run-api-check">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=run-api-check" />
+  </Target>
+  <Target Name="build-pack">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=build-pack" />
+  </Target>
+  <Target Name="native-compile">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=native-compile" />
+  </Target>
+  <Target Name="nuget-verify">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=nuget-verify" />
+  </Target>
+  <Target Name="nuget-install">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=nuget-install" />
+  </Target>
+  <Target Name="xunit-test">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=xunit-test" />
+  </Target>
+  <Target Name="make-roslyn-fast">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=make-roslyn-fast" />
+  </Target>
+  <Target Name="resx">
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="ExecShade" Properties="ShadeTarget=resx" />
+  </Target>
+  <Target Name="--no-restore">
+    <PropertyGroup>
+      <NoRestore>true</NoRestore>
+    </PropertyGroup>
+  </Target>
+  <Target Name="--quiet">
+    <PropertyGroup>
+      <MakeQuiet>true</MakeQuiet>
+    </PropertyGroup>
+  </Target>
+  <Target Name="--parallel" />
+
+  <Target Name="ExecShade">
+    <PropertyGroup>
+      <ShadeDirectory>$(KoreBuildDirectory)shade/</ShadeDirectory>
+      <SakeExecutable>$(RepositoryRoot)$(KoreBuildDirectory)sake/0.2.2/tools/Sake.exe</SakeExecutable>
+      <SakeExecutable Condition="'$(OS)'!='Windows_NT'">mono $(SakeExecutable)</SakeExecutable>
+      <MakeFilePath>$(ShadeDirectory)makefile.shade</MakeFilePath>
+      <MakeFilePath Condition="!Exists('$(RepositoryRoot)makefile.shade')">$(RepositoryRoot)makefile.shade</MakeFilePath>
+
+      <ShadeTarget Condition="'$(NoRestore)'=='true'">--no-restore $(ShadeTarget)</ShadeTarget>
+      <ShadeTarget Condition="'$(MakeQuiet)'=='true'">--quiet $(ShadeTarget)</ShadeTarget>
+    </PropertyGroup>
+
+    <Exec 
+      Command="$(SakeExecutable) -I $(KoreBuildDirectory)shade -f $(MakeFilePath) $(ShadeTarget)"
```

Comment:
`KoreBuildDirectory` is root relative - notice I had to concat the path on Line 145 with `RepositoryRoot` to make it absolute. Btw, this brought up an interesting issue with using back slashes in our code - the things we invoke on xplat don't like back slashes (for instance `mono build\sake.exe`). Calling `Path.GetFullPath` helps, but it'd be much better to standardize on forward slashes in our targets.